### PR TITLE
[INLONG-5192][DataProxy] Add unit test for TubeMQ sink

### PR DESCRIPTION
**Pull request by liangyepianzhou**
### Prepare a Pull Request
- Fixes https://github.com/apache/inlong/issues/5192

### Motivation

Add test for tube sink. KafkaSink has not been implemented yet, so test TubeSink first.

### Modifications

Add test for tube sink. KafkaSink has not been implemented yet, so test TubeSink first.

### Verifying this change

*(Please pick either of the following options)*

- [x] This change added test

### CODE REVIEW

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.apache.flume.event.EventBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestTubeSink {
+
+    private MemoryChannel channel;
+
+    @Before
+    public void setUp() throws Exception {
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:34]: The exception was not necessary throws here.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        channel = new MemoryChannel();
+        Context context = new Context();
+        context.put("type", "org.apache.inlong.dataproxy.sink.TubeSink");
+        sink.setChannel(channel);
+
+        this.channel.configure(context);
+    }
+
+    @Test
+    public void testProcess() throws Exception {
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:46]: ditto.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestTubeSink {
+
+    private MemoryChannel channel;
+
+    @Before
+    public void setUp() throws Exception {
+
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:35]: This blank line was not necessary too.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        Context context = new Context();
+        context.put("type", "org.apache.inlong.dataproxy.sink.TubeSink");
+        sink.setChannel(channel);
+
+        this.channel.configure(context);
+    }
+
+    @Test
+    public void testProcess() throws Exception {
+
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:47]: This blank line was not necessary too.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        channel = new MemoryChannel();
+        Context context = new Context();
+        context.put("type", "org.apache.inlong.dataproxy.sink.TubeSink");
+        sink.setChannel(channel);
+
+        this.channel.configure(context);
+    }
+
+    @Test
+    public void testProcess() {
```
Comment (baomingyu) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:45]: What is the goal of this unit test?

Comment (liangyepianzhou) [reply inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/TestTubeSink.java:45]: To test use TubeSink to process messages, just like TestPulsarSink.

**Review by healchow — APPROVED** [review]

**Comment by dockerzhang** [comment]
@liangyepianzhou please format the `TestPulsarSink` at the same time according to the comments, thanks.

**Comment by liangyepianzhou** [comment]
> @liangyepianzhou please format the `TestPulsarSink` at the same time according to the comments, thanks.

I have already fixed the comments given by [healchow](https://github.com/healchow).
What comments do you mean?

**Comment by dockerzhang** [comment]
Yes, the new file `TestTubeSink.java` you have fixed, I mean you can fix `TestPulsarSink` according to the comments given by [healchow](https://github.com/healchow) at the same time, that means we do not need another PR to fix formats issues in `TestPulsarSink`.

**Review by dockerzhang — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]
+1

**Pull request merged**